### DISCUSSION TITLE
Limit list document

### DISCUSSION THREAD

**Discussion by Raiper34** [appwrite/appwrite#5812] [Q&A]
Hi, 
I am new to appwrite, but it looks like very good technology, but still, I am confused with some things like. 
I create document and give rights only to specific user to read.
```
databases.createDocument('64aec7795b81cb50edac', '64aec7800f2f1b3ad15a', ID.unique(), {id: 5000}, [Permission.read(Role.user('64aecb25a0d01953de7f'))])
```
but when I list documents with another user I can see everything
```
databases.listDocuments('64aec7795b81cb50edac', '64aec7800f2f1b3ad15a')
```
Yes, I know about the query, but still, this is not much secure, I am sure I am missing something.

**Comment by rafagazani** [comment]
how is the permission of your collection?

**Comment by eldadfux** [comment]
You need to make sure you’ve enabled document level security.

**Comment by Raiper34** [comment]
Hah, you are right, I had got the wrong permission on the whole collection. The user could read all documents. I set up document-level permissions during document creation. Thanks for help and for awesome technology!
